schema/gks-common/common-source.yaml

```diff
@@ -1,20 +1,20 @@
 $schema: "https://json-schema.org/draft/2020-12/schema"
-$id: "https://w3id.org/ga4gh/schema/gks-common/1.x/core-source.yaml"
-title: GKS Common Library Core Definitions
+$id: "https://w3id.org/ga4gh/schema/gks-common/1.x/common-source.yaml"
+title: GKS Common Library Definitions
```

Comment:
Did we want to keep core?

Reply:
not here. Matt only want's to use the term `core` in reference to the `core-im` in va-spec.